### PR TITLE
RFC 35: Add third-party GitHub App: Pull Panda

### PR DESCRIPTION
**Pull request by zcorpan**
[View the rendered version](https://github.com/web-platform-tests/rfcs/blob/zcorpan/rfc-pullpanda/rfcs/pullpanda_analytics.md).

### CODE REVIEW

**Comment by jgraham** [comment]
Enabling a GH-provided app seems fine to me. Argualy it doesn't even need an RFC, just agreement from the web-platform-tests/wpt admins.

File: rfcs/pullpanda_analytics.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# RFC 35: Add third-party GitHub App: Pull Panda
+
+## Summary
+
+At TPAC 2019 we set out some [priorities for 2020](https://docs.google.com/document/d/1gie7LFb6cAUfabY86MYuWM7m7ux_FaKhDkLdpz0zWkg/edit),
+which include increasing PR review velocity and reduce the number of stale PRs.
+
+In order to track progress over time,
+install the [Pull Panda](https://pullpanda.com/) GitHub App
+for the web-platform-tests/wpt repository (or all repos in the org).
```
Comment (zcorpan) [inline rfcs/pullpanda_analytics.md]: We should decide whether to enable it for only wpt or the entire org.

Comment (foolip) [reply rfcs/pullpanda_analytics.md]: If it doesn't create statuses, deployments or anything like that, then enabling it for the whole org seems fine.

**Comment by zcorpan** [comment]
I've clarified that we intend to enable this for the whole org (rather than only the wpt repo).

**Comment by foolip** [comment]
@zcorpan given that it's read-only, can you go ahead and enable it so we can see it live? This RFC can remain open for a week and we should turn it off if anyone raises concerns within that week.

**Comment by zcorpan** [comment]
Done. The app can be configured and uninstalled here:

https://github.com/organizations/web-platform-tests/settings/installations/4993614

Go to https://pullpanda.com/ and log in with your GitHub account. Choose "web-platform-tests" and continue to "Analytics". Click "Analytics" at the top again to view different graphs.

It probably takes a while for it to have all the data for some of the graphs.

**Comment by zcorpan** [comment]
I noticed now that the history is 24 weeks (almost 6 months), rather than 3 months.

**Comment by zcorpan** [comment]
Hmm. I wonder if http://github.com/web-platform-tests/wpt-actions-test/pull/17 broke Pull Panda. The "Longest time to merge" list includes only that PR, and some other graphs are missing data.

**Comment by zcorpan** [comment]
I've changed the selected repos to wpt, wpt-pr-bot, wpt.fyi and rfcs, but I think the problem persists, maybe due to https://github.com/web-platform-tests/wpt/pull/20036 which also has an empty commit.

**Comment by foolip** [comment]
Funny how we seem to be able to break other tools with all the weird things we get up to in this repo :)

**Comment by foolip** [comment]
Enough time has passed, I'll merge this now.

**Comment by zcorpan** [comment]
I've filed another issue for Pull Panda about the lack of ability to filter away bot-created PRs in the analytics. https://github.com/pullreminders/backlog/issues/210

**Pull request merged**